python/meep.i

```diff
@@ -372,6 +372,7 @@ PyObject *_get_dft_array(meep::fields *f, dft_type dft, meep::component c, int n
     }
 
     PyObject *py_arr = PyArray_SimpleNewFromData(rank, arr_dims, NPY_CDOUBLE, dft_arr);
+    PyArray_ENABLEFLAGS((PyArrayObject*)py_arr, NPY_ARRAY_OWNDATA);
```

Comment:
This will eventually cause [NumPy to call `free`](https://github.com/numpy/numpy/blob/31f19645a13ee221f037ade57f13006127aa4c46/numpy/core/src/multiarray/alloc.c#L257-L274), so it is not safe to call with an array allocate with `new` like `get_dft_array.

Reply:
It would be nice to be able to get the array size and allocate a numpy array from Python like we do in `Simulation.get_array`, but I don't see a way to get the dft array size, so I'm copying the dft data to my own `malloc`ed array.

Reply:
I don't think you need to set the `NPY_ARRAY_OWNDATA` flag if you are using `PyArray_SimpleNew`